nyholm/psr7/0.3/config/packages/nyholm_psr7.yaml

```diff
@@ -0,0 +1,32 @@
+# Read more about client options at: https://github.com/kriswallsmith/Buzz/blob/master/doc/client.md
```

Comment:
this doc link looks wrong. It goes to a different package.

Reply:
Thank you. Copy/paste error